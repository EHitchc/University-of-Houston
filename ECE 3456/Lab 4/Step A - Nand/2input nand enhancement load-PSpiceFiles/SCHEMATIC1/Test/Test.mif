 lib=, offset=43, size=41
