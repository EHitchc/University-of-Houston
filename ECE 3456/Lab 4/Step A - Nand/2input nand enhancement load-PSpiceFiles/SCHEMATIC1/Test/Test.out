
**** 11/14/18 18:42:15 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ c:\users\ethan\documents\ece 3456\lab 4\step a - nand\2input nand enhancement load-PSpiceFiles\SCHE


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../2input nand enhancement load-PSpiceFiles/2INPUT NOR ENHANCEMENT LOAD.lib" 
* From [PSPICE NETLIST] section of C:\Users\Ethan\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source 2INPUT NAND ENHANCEMENT LOAD
.EXTERNAL INPUT Vout
M_M1         N00207 N00207 VOUT VOUT MbreakN           
M_M2         VOUT N00229 N00130 0 Mbreakn           
M_M3         N00130 N00238 0 0 MbreakN           
V_A         N00229 0 0
V_B         N00238 0 5
V_Vdd         N00207 0 5Vdc

**** RESUMING Test.cir ****
.END

**** 11/14/18 18:42:15 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ c:\users\ethan\documents\ece 3456\lab 4\step a - nand\2input nand enhancement load-PSpiceFiles\SCHE


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               MbreakN         
               NMOS            
       LEVEL    1            
           L  100.000000E-06 
           W  100.000000E-06 
         VTO     .91         
          KP  685.600000E-06 
       GAMMA    0            
         PHI     .6          
      LAMBDA    0            
          IS   10.000000E-15 
          JS    0            
          PB     .8          
        PBSW     .8          
          CJ    0            
        CJSW    0            
        CGSO    0            
        CGDO    0            
        CGBO    0            
         TOX    0            
          XJ    0            
       UCRIT   10.000000E+03 
      DIOMOD    1            
         VFB    0            
        LETA    0            
        WETA    0            
          U0    0            
        TEMP    0            
         VDD    5            
       XPART    0            


**** 11/14/18 18:42:15 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ c:\users\ethan\documents\ece 3456\lab 4\step a - nand\2input nand enhancement load-PSpiceFiles\SCHE


 ****     SMALL SIGNAL BIAS SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


( VOUT)    4.0899  (N00130)-15.15E-27 (N00207)    5.0000 (N00229)    0.0000     

(N00238)    5.0000 




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_A          0.000E+00
    V_B          0.000E+00
    V_Vdd       -4.100E-12

    TOTAL POWER DISSIPATION   2.05E-11  WATTS



          JOB CONCLUDED

**** 11/14/18 18:42:15 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ c:\users\ethan\documents\ece 3456\lab 4\step a - nand\2input nand enhancement load-PSpiceFiles\SCHE


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .02
